addons/point_of_sale/static/src/app/models/related_models.js

```diff
@@ -24,6 +24,10 @@ function mapObj(obj, fn) {
     return Object.fromEntries(Object.entries(obj).map(([k, v], i) => [k, fn(k, v, i)]));
 }
 
+function equalRecords(a, b) {
```

Comment:
should be interesting to add this to the Base class, to be able to call it when comparing record

```js
product.isEqual(product)
```

what do you think ?